meta/internal/meta.proto

```diff
@@ -0,0 +1,237 @@
+package internal;
+
+//========================================================================
+//
+// Metadata
+//
+//========================================================================
+
+message Data {
+	repeated NodeInfo Nodes = 1;
+	repeated DatabaseInfo Databases = 2;
+}
+
+message NodeInfo {
+	required uint64 ID = 1;
+	required string Host = 2;
+}
```

Comment:
I think so.  Otherwise a caller wouldn't know which nodes to hit and an AP node wouldn't know where to redirect a remote call.